docs/source/conf.py

```diff
@@ -67,6 +67,9 @@
         'searchbox.html',
     ]
 }
+html_theme_options = {
```

Comment:
Because the default value is 4, a comment describing why 3 works better would be great.

Reply:
just the question of looks :D Fixed